games/Hangman.py

```diff
@@ -0,0 +1,116 @@
+import random
+
+# simple terminal hangman game with ASCII art
+def choose_random_word():
```

Comment:
Trivial: Can we add docstrings to functions, for example:

```
def choose_random_word():
    """Chooses a random word between the ones listed in word_List
        Returns: (str) the chosen word.
    """
    word_list = ["hangman", "python", "programming", "developer", "challenge", "knowledge"]
    return random.choice(word_list)
```

Reply:
Just added docstrings in my latest commit :)